SECURITY.md

```diff
@@ -14,3 +14,14 @@ If you have found or just suspect a security problem somewhere in curl or
 libcurl, report it on [HackerOne](https://hackerone.com/curl).
 
 We treat security issues with confidentiality until controlled and disclosed responsibly.
+
+## OpenSSF Scorecard
+
+curl has earned Gold status on the OpenSSF Scorecard, reflecting its adherence to
```

Comment:
The [OpenSSF scorecard](https://github.com/ossf/scorecard) is different from the [OpenSSF best practices](https://www.bestpractices.de). The link you provide here is for the best practices badge. Worth getting the terminology right I think!

Reply:
My bad! Yes, Scorecard is something different. Got it confused in my head. Pushed a fix.